spice_ev/events.py

```diff
@@ -247,7 +248,7 @@ def get_schedule_from_csv(obj, dir_path):
         window_col_idx = header.index(window_col) if window_col in header else None
 
         if obj.get('individual'):
-            vehicle_names = header[3:]
+            vehicle_names = header[7:]
```

Comment:
Is this and line 301-302 because `[kW]` is added to the vehicle_names?

Reply:
No. Curtailment and residual load are added to schedule CSV and number of columns change. The last columns contain the vehicle data. That is why other column numbers have to be chosen.